site/content/blog/2020-10-01-whats-new-in-svelte-october-2020.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What's new in Svelte: October 1, 2020
+description: New object methods, in-depth learning resources and tons of integration examples!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
```

Comment:
Maybe we could add some short introductionary sentences here, like "it was quiet for some time but we promise to be more active with news from now on".